GuessingGame/Service.cs

```diff
@@ -11,9 +11,9 @@ public Service(INumberGenerator numberGenerator)
         {
             _numberGenerator = numberGenerator;
         }
-        
```

Comment:
In Production code you wouldn't name a class just Service, it doesn't clearly explain what it does, when you catch up to SOLID principles you will see that S stands for SRP or single responsibility Principle. If you name a class too generic then its difficult for other developers to know what the intention of the class is and they would end up adding more code to it, which would bloat the service too much.